diff --git a/node_modules/expo-dev-menu/ios/DevMenuViewController.swift b/node_modules/expo-dev-menu/ios/DevMenuViewController.swift
--- a/node_modules/expo-dev-menu/ios/DevMenuViewController.swift
+++ b/node_modules/expo-dev-menu/ios/DevMenuViewController.swift
@@ -61,7 +61,11 @@ class DevMenuViewController: UIViewController {
   // MARK: private
 
   private func initialProps() -> [String: Any] {
-    let isSimulator = TARGET_IPHONE_SIMULATOR > 0
+    #if targetEnvironment(simulator)
+    let isSimulator = true
+    #else
+    let isSimulator = false
+    #endif
     
     return [
       "showOnboardingView": manager.shouldShowOnboarding(),
diff --git a/node_modules/expo-dev-menu/expo-dev-menu.podspec b/node_modules/expo-dev-menu/expo-dev-menu.podspec
--- a/node_modules/expo-dev-menu/expo-dev-menu.podspec
+++ b/node_modules/expo-dev-menu/expo-dev-menu.podspec
@@ -70,7 +70,6 @@ Pod::Spec.new do |s|
 
     # Swift/Objective-C compatibility
     safearea.pod_target_xcconfig = {
-      'DEFINES_MODULE' => 'YES',
       'SWIFT_COMPILATION_MODE' => 'wholemodule'
     }
     if File.exist?("vendored/react-native-safe-area-context/dev-menu-react-native-safe-area-context.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
